.circleci/unittest/linux/scripts/setup_env.sh

```diff
@@ -21,10 +21,9 @@ esac
 # 1. Install conda at ./conda
 if [ ! -d "${conda_dir}" ]; then
     printf "* Installing conda\n"
-    wget --quiet -O miniconda.sh "http://repo.continuum.io/miniconda/Miniconda3-latest-${os}-x86_64.sh"
+    wget --quiet -O miniconda.sh "http://repo.continuum.io/miniconda/Miniconda3-py37_4.10.3-${os}-x86_64.sh"
     bash ./miniconda.sh -b -f -p "${conda_dir}"
     eval "$("${conda_dir}/bin/conda" shell.bash hook)"
-    conda update --quiet -y conda
```

Comment:
Actually, removing this line should be sufficient

Reply:
Sure. Let me revert and test it.